### ISSUE TITLE
Dude.

### ISSUE THREAD

**Issue by purzbeats** [kijai/ComfyUI-PyramidFlowWrapper#1]
You're too fast. 💖

**Comment by kijai** [comment]
Oh man I wish...... this one has been causing me some gray hairs all day...

But I did just run 1280x768 first time succesfully under 20GB, there's hope!

**Comment by kijai** [comment]
Under 12GB actually:

![image](https://github.com/user-attachments/assets/b8fc3298-daa3-486b-9458-315a34d8eef7)

**Comment by purzbeats** [comment]
I got it running locally last night but it OOM'd on the 384p model, they added VAE tiled decoding I think last night so I'm gonna try again 🤣

**Comment by kijai** [comment]
Yeah that VAE took a lot, but it's also that their pipeline had no offloading of text_encoder etc. at all.

**Comment by al3dv2** [comment]
Does it works with 24go of vram with the vae ? thanks

**Comment by al3dv2** [comment]
I just tried and the vae cause a OOM even with 24go of vram

**Comment by lijackcoder** [comment]
YOOOO I just got my new 16gb vram, i could run this !?!?! Can't wait for the update
